app.js

```diff
@@ -7,6 +7,13 @@ const govukTemplate = require('hof-govuk-template');
 const hoganExpressStrict = require('hogan-express-strict');
 const expressPartialTemplates = require('express-partial-templates');
 const _ = require('lodash');
+const apiCall = require('./lib/api-call');
+const config = require('./config');
+const baseUrl = config.apiBaseUrl;
```

Comment:
if you *have* to do this rather than refer to `config.var` then consider object destructuring